### ISSUE TITLE
[XamarinProfiler-MAC]: Usability : VoiceOver : The chart information under the 'Top Allocations' triangular button of the 'Inspect' tab is not announced by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#434]
### Prerequisites

1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application."
6. Navigate to the "Choose Target" button of the Xamarin Profiler window.

### Steps to Reproduce

1. Follow the pre-requisites to launch the Xamarin Profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button. Invoke Start Profiling button.
3.This will invoke "Allocations/Time Profiler" Profiling page.
4. Navigate to Allocations -> Allocations Tab and invoke the Tab.
5. Navigate to the Inspect tab on the right hand side of the page.
6. Navigate to the 'Top Allocations' triangular button and observe the behavior of the VoiceOver for the chart .

### Expected Behavior

The chart information under the 'Top Allocations' group should be announced properly by the VoiceOver.

### Actual Behavior

The VoiceOver simply announces the name of the 'Top Allocations' triangular button but no information about the chart  is announced. 

### User Impact

The Screen-reader users won't be able to know the presence of the chart under the 'Top Allocations' group. 

### Screenshots

![voiceover_profilermac7](https://user-images.githubusercontent.com/22956130/41714323-8e767090-756d-11e8-9c4d-093a8149dbac.png)

### Environment

macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Usability
Xamarin-Profiler-MAC
Wipro
